src/framework/components/scroll-view/data.js

```diff
@@ -1,6 +1,10 @@
+import { Vec2 } from '../../../math/vec2.js';
+
 class ScrollViewComponentData {
     constructor() {
         this.enabled = true;
+        this.useMouseWheel = true;
+        this.mouseWheelSensitivity = new Vec2(1, 1);
```

Comment:
Check out this issue:

https://github.com/playcanvas/engine/issues/1332

In other words, new component properties should live on the component, not the soon-to-be removed `ComponentData` classes.

Reply:
Not sure I understand this. I've added the new properties to `const _schema_`, which, following the code in `initializeComponentData()`, is the unified location of the component's properties. It looked to me, then, that `ScrollViewComponentData` is for storing the 'default' values for those properties.

Do you then mean that I don't need to put anything under `ScrollViewComponentData`, and instead set the default values directly in the component's constructor?

Reply:
Yeah, pretty much. Look at how I ported the RigidBody component from the old format to the new one:

https://github.com/playcanvas/engine/pull/3164